plugin.py

```
@@ -464,30 +463,62 @@ def job(m: Movie) -> None:
+
+            LOGGER.warning(
+                "Provider %s did not return episodes, trying next provider",
+                relation.m3u_account.name
+            )
+
+        LOGGER.warning(
+            "No provider returned episodes for series_id=%s after trying %d provider(s)",
+            series.id,
+            relations.count()
+        )
         return False
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)